vtex/utils/transform.ts

```diff
@@ -969,3 +972,14 @@ export const toReview = (
     };
   });
 };
+
+export const sortProducts = (
```

Comment:
We can do a O(n) sort instead of a O(n^2log(n)) with an insertion sort like algorithm

Reply:
Can you check now?